ios/StripeSdkImpl.swift

```diff
@@ -206,7 +206,7 @@ public class StripeSdkImpl: NSObject, UIAdaptivePresentationControllerDelegate {
                                     rejecter reject: @escaping RCTPromiseRejectBlock) -> Void  {
         DispatchQueue.main.async {
             if (self.paymentSheetFlowController != nil) {
-                self.paymentSheetFlowController?.confirm(from: UIApplication.shared.rootViewControllerWithFallback()) { paymentResult in
+                self.paymentSheetFlowController?.confirm(from: RCTKeyWindow()?.rootViewController ?? UIViewController()) { paymentResult in
```

Comment:
Incoming changes from #2216 changed how we're getting our root view controller. I had previously commonized the logic on this branch, but to prevent future conflicts, I’m following patterns from `master` to reduce future conflicts in areas not related to crypto onramp.